code/lib/resampleNifti.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import nibabel as nb
+
+from nilearn.image import resample_img
+
+
+def resampleNifti(img, resampling_factor, filename):
+
+    nii = nb.load(img)
+
+    dimensions = nii.header.get_data_shape()
+
+    vox_size = nii.header.get_zooms()
+
+    new_voxel_size = vox_size[1] / resampling_factor
```

Comment:
I think this assumes that voxels are isotropic, no?

I guess we could have the function, either accept a resampling_factor, or directly have the user specify which final resolution they want.